rep-2004.rst

```diff
@@ -375,54 +306,43 @@ These packages are more lax on documentation, testing, and scope of public API's
-
 Requirements to be considered a 'Level 3' package:
 
-* Version Policy:
-
-  * The same as 'Level 1' packages, except:
-
-    * No public API needs to be explicitly declared, though this can make it harder to maintain API and ABI stability
-    * No requirement to keep API/ABI stability within a stable ROS release, but it is recommended still
+#. **Version Policy:**
 
-* Change Control Process:
+   #. The same as 'Level 1' packages, except:
 
-  * Must have all code changes occur through a change request (e.g. pull request, merge request, etc.)
-  * Must have Continuous Integration (CI) policy for all change requests
-  * [ROS Core]:
+      #. *No public API needs to be explicitly declared, though this can make it harder to maintain API and ABI stability*
+      #. *No requirement to keep API/ABI stability within a stable ROS release, but it is recommended still*
 
-    * All changes will go through a pull request
-    * All pull requests will be tested via CI
+#. **Change Control Process:**
 
-* Documentation:
+   #. Must have all code changes occur through a change request (e.g. pull request, merge request, etc.)
+   #. Must have Continuous Integration (CI) policy for all change requests
 
-  * Must have a declared license or set of licenses
-  * Must have a copyright statement in each source file
-  * May have a "quality declaration" document, which declares the quality level and justifies how the package meets each of the requirements
+#. **Documentation:**
 
-    * Must have a section in the repository's ``README`` which contains the "quality declaration" or links to it
-    * May register with a centralized list of 'Level 3' packages, if one exists, to allow for peer review of the claim
+   #. Must have a declared license or set of licenses
+   #. Must have a copyright statement in each source file
+   #. May have a "quality declaration" document, which declares the quality level and justifies how the package meets each of the requirements
 
-  * [ROS Core]:
+      #. *Must have a section in the repository's ``README`` which contains the "quality declaration" or links to it*
+      #. *May register with a centralized list of 'Level 3' packages, if one exists, to allow for peer review of the claim*
 
-    * Must have automated checks for copyright statements and licenses
-    * Must use the Apache 2.0 license, unless the package has an existing permissive license (e.g. rviz uses three-clause BSD)
+#. **Testing:**
 
-* Testing:
+   #. No explicit testing requirements, though covering some if not all of the features with tests is recommended
 
-  * No explicit testing requirements, though covering some if not all of the features with tests is recommended
+#. **Dependencies:**
 
-* Dependencies:
+   #. May have direct runtime "ROS" dependencies which are not 'Level 3' dependencies, but they should be documented
```

Comment:
level 3's dependencies section is the only one that mentions that dependencies should be documented (making it difficult to give it a ✓ or ● compared to level 1).

should the fact that dependencies and all their caveats and justifications need to be documented be added to level 1's dependencies section as a standalone requirement (`5.iv`) or to level 1's (and all subsequent levels) documentation section (`3.v Must have a "quality declaration" document, which declares the quality level and justifies how the package AND ITS DEPENDENCIES meet each of the requirements?`)?

Reply:
By "documented" I think I meant, "documented as part of the quality declaration". So I don't think it needs updating in the documentation requirements. Instead I'd suggest making this point more specific, that we mean the dependencies should be considered and covered by the quality declaration.